packages/vue-dot/src/patterns/FooterBar/CollapseList/CollapseList.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div class="vd-collapse-list my-4">
+		<div v-if="!isMobile">
+			<div
+				class="mb-2"
+			>
+				<span class="text-subtitle-1 font-weight-bold">
+					{{ listItems.title }}
+				</span>
+			</div>
+			<div
+				v-for="item in listItems.list"
+				:key="item.name"
+				class="mb-2"
+			>
+				<a
+					:href="item.route"
```

Comment:
```suggestion
					:href="item.href"
```
Avec `route` je m'attends plutôt à un objet à passer à Vue Router